pages/tools.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Tools
```

Comment:
>Tools [](http://example.com/codeflow?start=7&length=5)

this page is currently blank. What happened to the content from the original page?

We should either add content or hide this page.

Reply:
Maybe we can add links to [http://rapid.rocks/api-designer/index.html](http://rapid.rocks/api-designer/index.html) and [http://rapid.rocks/api-explorer/index.html](http://rapid.rocks/api-explorer/index.html)